data/remnant outfits.txt

```diff
@@ -449,6 +449,18 @@ outfit "Tuning Rifle"
 	"unplunderable" 1
 	description "The Tuning Rifle uses sound waves to destabilize the weaker internal tissue of many races and liquefies them."
 
+outfit "Void Rifle"
+	category "Special"
+	licenses
+		Remnant
+	cost 1000000
+	thumbnail "outfit/void rifle"
+	"capture attack" 20
+	"capture defense" 20
+	"unplunderable" 1
+	description "How the Void Rifle works isn't clearly understood, but when fired it emits a form of non-visible radiation that has a strong paralyzing effect on the nervous system. Given that it has an area of effect that penetrates walls and other solid objects, it is an incredibly effective weapon."
+	description "The Remnant made a significant number of these weapons during the early years of their research on the Void Sprites, but haven't been able to make any new ones since the Archon stopped their research activities. While durable, they do rarely last more than a few years, so even the best cared for of the old void rifles degraded into uselessness centuries ago. These highly effective weapons are credited with helping the Remnant resist Korath invaders on the ground before they had ships capable of defending their skies."
```

Comment:
I made changes for most of those.

In regards to making it more explicit that these are chunks of void sprite tentacle, how explicit do you think I should be? I changed the previous sentence slightly to state that the reason they stopped making them was because the Archon stopped their harvests (as opposed to research activities), so that'll be a bit more direct. Do you think I should go farther?

And yes, it looks like I need to review my timeline for the Korath and ships. I like the description and implications for planet-side history, but that does throw other aspects of their history into confusion.

Either this description needs to change, or the previously described time-span for the Korath conflict needs to start several centuries earlier. I'm not sure which one makes for better story, although having a longer war does make their reverse-engineering of Korath stuff more reasonable. (for those who don't like the implications that they do fast reverse engineering)

Reply:
OK, I changed the Void Rifle's description so that it doesn't specifically mention the Korath, opting instead for:
`These highly effective weapons are credited with helping the Remnant safeguard their refuges from planetary threats during their early years.`